src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -282,11 +282,17 @@ const ContentListTable = () => {
                     },
                   },
                   {
-                    isDisabled: actionTakingPlace || rowData?.status === 'Retrying',
+                    isDisabled:
+                      actionTakingPlace || rowData?.status === 'Retrying' || !rowData.snapshot,
                     title: 'Trigger snapshot',
                     onClick: () => {
                       triggerSnapshot(rowData.uuid);
                     },
+                    tooltipProps: {
+                      content: 'Snapshots disabled for this repository.',
+                      isVisible: !rowData.snapshot,
+                      trigger: 'manual',
+                    },
```

Comment:
So I tested this out, and although it will present the tooltip, the tooltip isn't being triggered via mouseenter / focus. 
So when I open the menu the tooltip would appear. 

I played around with this and found that you can use the "triggerRef" to attach the tooltip to the document programmatically. 

You can also use the !rowData.snapshot bool to conditionally add that value to the row. 

Here's what I ended up with.

`
tooltipProps: !rowData.snapshot
  ? {
      content: 'Snapshots disabled for this repository.',
      triggerRef: () =>
        document.getElementById('actions-column') || document.body,
    }
  : undefined,
`
`
   <ActionsColumn id='actions-column' items={rowActions(rowData)} />
`

Reply:
oh yep you're right, i was moving my mouse too fast so i thought it was just always happening on mouseenter XD will try this, thank you!